INSTALL_DOCKER.rst

```diff
@@ -35,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
 
 This change will take effect once you have logged out as root and logged back in as <USERNAME>.
```

Comment:
right. Maybe it would be less confusing all round to remove references to newgrp and just go for the logout/in approach.